main.go

```diff
@@ -14,16 +12,25 @@ const VERSION = "0.0.1"
 
 func main() {
 	log.Logf("Started v%s", VERSION)
-	// Start the web server
-	godotenv.Load()
 
-	if os.Getenv("DB_ACTIVE") == "true" {
-		model.InitDB()
-		controller.InitController()
+	// Load .env file
+	err := godotenv.Load()
+	if err != nil {
+		log.Errf("Error loading .env file: %s", err)
+		return
+	}
+
+	// Initialize the controllers with the database
+	err = controller.InitController()
```

Comment:
Since this is the only initialization in the controller package, we can forgo the redundant `Controller` in `InitController`